module-theme-functional-checklist/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Module / theme functional checklist
+weight: 6
+pre: "<b>6. </b>"
+chapter: true
+---
+
+### Chapter 6
+
+# Module / theme functional checklist
+
+The functional testing checklist includes prerequisites, activities, resources, etc. in a sequential and guided way to cover the need to satisfy the functional requirements in all possible dimensions.
+
+### Compatibility:
+Your module/theme should be compatible with [PHP and Prestashop compatibility chart]({{< ref "1.7/basics/installation/system-requirements/#php-compatibility-chart" >}})
+
+###Configure button:
+Display the quick option button after installing the module from the Module catalog page. 
+
+###Error / Success message:
+* Display a confirmation or an error alert when the merchant saves a form. 
+* Display an error message when the merchant leaves the obligatory fields empty.
+* Display an error message when the merchant chooses the wrong file type.
+
+Example: a PDF file instead of a picture (display an error message when the merchant puts a wrong input value).
+
+* Display an error message when the merchant puts a wrong input value.
+
+###POP-UPS:
+* Testing all text pop-ups.
+* Verification of the messages for updating and deleting.
+* Text messages on errors in case of incorrect input.
+
+###Inputs:
+All input types must be checked before saving them. Be sure that:
+* You can only put numbers in a numeric field.
+* Email input must accept only email format.
+* The real number must include both rational and irrational numbers.
+
+###Social Networks:
```

Comment:
@mariem-abid this definitely doesn't apply to any best practices, etc. It's Addons Marketplace specific thing